docs/tools/vdb_table/data/rockset.json

```diff
@@ -83,28 +83,28 @@
     "comment": ""
   },
   "rag": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/sofia099/vector_search_demo",
```

Comment:
This column is to show whether the VDB has an in-built way of calling an LLM API (from within the DB), to do RAG over the retrieved documents.

Example: https://weaviate.io/developers/weaviate/modules/reader-generator-modules/generative-openai